packages/core/content-releases/server/src/services/release.ts

```diff
@@ -441,43 +443,80 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
 
     /**
      * We separate publish and unpublish actions group by content type
+     * And we keep only their ids to fetch them later to get all information needed
      */
     const actions: {
       [key: UID.ContentType]: {
-        publish: ReleaseAction['entry'][];
-        unpublish: ReleaseAction['entry'][];
+        actionsToPublishIds: ReleaseAction['entry']['id'][];
+        actionsToUnpublishIds: ReleaseAction['entry']['id'][];
```

Comment:
I think we should be careful about the wording. We are publishing entries on the action so I think we should refer to these as `entriesToPublishIds` and I would review the other variable names as well.